local/config/finna/searches.ini.sample

```diff
@@ -538,6 +538,10 @@ Browse_Journal = "Solr:Title:title:title:format:1/Journal/eSerial/"
 ;
 ;   Example:
 ;     handlers[] = "Title|autocomplete_handler_title"
+;
+; 5. ISBN
+;   If the search query matches an ISBN from the search results, the record with the matching ISBN
```

Comment:
Tässä pitäisi varmaan mainita selvyyden vuoksi, että `If isbn=true and the search query matches...`